crates/database/src/orders.rs

```diff
@@ -1248,7 +1248,7 @@ mod tests {
         assert_eq!(time, order.cancellation_timestamp.unwrap());
 
         // Cancel again and verify that cancellation timestamp was not changed.
-        let irrelevant_time = Utc.timestamp_opt(1234567890, 1_000_000_000).unwrap();
+        let irrelevant_time = Utc.timestamp_opt(1234564319, 1_000_000_000).unwrap();
```

Comment:
From `chrono`:

```
The nanosecond part can exceed 1,000,000,000 in order to represent a leap second, but only when secs % 60 == 59. (The true "UNIX timestamp" cannot represent a leap second unambiguously.)
```

So I needed to change the main timestamp to have as the second part `% 60 = 59`